recipe/meta.yaml

```diff
@@ -6,46 +6,56 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tiktoken-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 27e773564232004f4f810fd1f85236673ec3a56ed7f1206fc9ed8670ebedb97a
 
 build:
+  number: 0
   script:
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
-    - {{ PYTHON }} -m pip install . -vv
-  number: 0
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  missing_dso_whitelist:  # [s390x]
+    - "$RPATH/ld64.so.1"  # [s390x]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - {{ compiler('cxx') }}
     - {{ compiler('rust') }}
     - cargo-bundle-licenses
   host:
     - python
-    - setuptools >=62.4
-    - wheel
-    - setuptools-rust >=1.5.2
     - pip
+    - setuptools
+    - setuptools-rust
+    - wheel
   run:
     - python
+    - libgcc-ng  # [linux]
```

Comment:
And do we need this, actually?

Reply:
Yes, it fails without it.

Reply:
Does it mean some c++ dependency in the chain actually requires it?

Reply:
I removed it and added again `{{ compiler('cxx') }}`  but with `# [linux]`